components/calendar/src/date.rs

```diff
@@ -236,13 +236,26 @@ impl<A: AsCalendar> Date<A> {
         self.calendar.as_calendar().year_info(&self.inner).into()
     }
 
-    /// The "extended year", typically anchored with year 1 as the year 1 of either the most modern or
-    /// otherwise some "major" era for the calendar
+    /// This currently returns the same value as [`Self::monotonic_year()`]
+    #[deprecated = "Please use monotonic_year() instead"]
+    #[inline]
+    pub fn extended_year(&self) -> i32 {
+        self.monotonic_year()
+    }
+
+    /// The "monotonic year", typically anchored with year 1 as the year 1 of either the most modern or
+    /// otherwise some "major" era for the calendar.
+    ///
+    /// This year number can be used when you need a simple numeric representation
+    /// of the year.
+    ///
+    /// For calendars defined in Temporal, this will match the "arithmetic year"
+    /// as defined in <https://tc39.es/proposal-intl-era-monthcode/>.
     ///
     /// See [`Self::year()`] for more information about the year.
     #[inline]
-    pub fn extended_year(&self) -> i32 {
-        self.calendar.as_calendar().extended_year(&self.inner)
+    pub fn monotonic_year(&self) -> i32 {
```

Comment:
Do we need this API? Callers can just call `.year().monotonic_year()` (maybe rename that to `year().monotonic()`?)